homework-assignments-1/solutions/task-4.ts

```diff
@@ -0,0 +1,200 @@
+class Product {
+  public name: string;
```

Comment:
make class members always private just in case you don't use it in derived classes. You should only interact with class members with getters and setters